src/views/HashtagPostView/index.tsx

```diff
@@ -0,0 +1,158 @@
+    // In this case post data will be the first post of the hashtag
+    if (!query.slug?.[2])
+      push(
+        `/${locale}/${pageData.hashtag?.page?.slug}/${pageData.hashtag?.slug}/${pageData.slug}`,
+      )
+  }, [])
+
+  return (
+    <Layout
+      seo={{
+        title: pageData?.text as string,
+        description: '',
+        image:
+          `${process.env.NEXT_PUBLIC_ADMIN_URL}${pageData?.image?.url}` as string,
+        url: `${process.env.NEXT_PUBLIC_SITE_URL}/${locale}/${pageData?.hashtag?.page?.slug}/${pageData?.hashtag?.slug}/${pageData?.slug}`,
+        width: pageData?.image?.width as number,
+        height: pageData?.image?.height as number,
+        type: pageData?.image?.mime as string,
+      }}
+    >
+      <Container py={4}>
+        <Box textAlign="center">
+          <Heading>{pageData?.hashtag?.title}</Heading>
+          <Text my={4} maxW="container.md" mx="auto">
+            {pageData?.hashtag?.content}
+          </Text>
+        </Box>
+        <Stack spacing={4} direction={{ base: 'column', lg: 'row' }}>
+          <Tabs flex={1} variant="soft-rounded" isFitted colorScheme="primary">
+            <TabList>
+              <Tab
+                color="gray.400"
+                fontWeight="bold"
+                borderBottomWidth={2}
+                _selected={{
+                  bg: 'primary.100',
+                  color: 'primary.400',
+                  borderColor: 'primary.400',
+                }}
+              >
+                {t`post-share.tabs.share`}
+              </Tab>
+              <Tab
+                color="gray.400"
+                fontWeight="bold"
+                borderBottomWidth={2}
+                _selected={{
+                  bg: 'primary.100',
+                  color: 'primary.400',
+                  borderBottomWidth: 2,
+                  borderColor: 'primary.400',
+                }}
+              >
+                {t`post-share.tabs.archive`}
+              </Tab>
+            </TabList>
+            <TabPanels>
+              <TabPanel px={0}>
+                <HStack
+                  justify="stretch"
+                  align="stretch"
+                  minH={0}
+                  maxH={{ base: 'min-content', lg: 650 }}
+                  spacing={4}
+                >
+                  <VStack
+                    w={300}
+                    align="stretch"
+                    display={{ base: 'none', lg: 'flex' }}
+                  >
+                    <MentionSearch />
+                    <MentionList />
+                    <TrendList />
+                  </VStack>
+                  <Drawer isOpen={isOpen} placement="left" onClose={onClose}>
+                    <DrawerOverlay />
+                    <DrawerContent>
+                      <DrawerCloseButton />
+                      <DrawerHeader>{t`post-share.add-mention`}</DrawerHeader>
+                      <DrawerBody>
+                        <MentionSearch />
```

Comment:
I think we should put one custom mentionList here for search populer users from twetter.